storybook/src/elements/nuxeo-data-table/nuxeo-data-table-structure.js

```diff
@@ -0,0 +1,57 @@
+import { html } from 'lit-html';
+import '../../../../ui/widgets/nuxeo-date-picker';
```

Comment:
We are importing `nuxeo-date-picker` here, but `nuxeo-date` is being imported on the story file, which is not consistent. I think that we should import them in the same place, and it would make sense to import them in this file, which is where they are actually used.